chart/values.yaml

```diff
@@ -466,6 +471,44 @@ rows:
     annotations: {}
   tolerations: []
 
+search:
+  # the path of the external authentication service on the hub.
+  # The string must contain `%s` which will be replaced with the dataset name.
+  hfAuthPath: "/api/datasets/%s/auth-check"
```

Comment:
I think we're repeating this a lot (api, rows, search). Could we factorize?

Reply:
I moved all hfX configurations to another section `hf`